### PR TITLE
logging test types/host names - e2e and stress tests

### LINKED ISSUE #5576: Real Service Test - Logging prop denoting which test is running
## Work Item
**Describe the outcome you expect**
Either e2e ODSP, e2e r11s, or stress test (not exact strings btw)

**Approach**
Use ChildLogger to set the field before running.

**Open questions**
Maybe don't put ODSP v. r11s in there since there's already a driverType field

### PR DESCRIPTION
**Pull request by chensixx**
Fixes #5576 
Added a property "hostName" to both e2e and stress test logging

### CODE REVIEW

File: packages/test/test-service-load/src/utils.ts [hunk 1]
```diff
@@ -96,7 +96,8 @@ export async function initialize(testDriver: ITestDriver, seed: number) {
         urlResolver: testDriver.createUrlResolver(),
         documentServiceFactory: testDriver.createDocumentServiceFactory(),
         codeLoader: createCodeLoader(random.pick(randEng, generateRuntimeOptions(seed))),
-        logger: ChildLogger.create(await loggerP, undefined, {all: { driverType: testDriver.type }}),
+        logger: ChildLogger.create(await loggerP, undefined, {all: { driverType: testDriver.type,
+                hostName: "Stress Tests" }}),
```
Comment (anthony-murphy) [inline packages/test/test-service-load/src/utils.ts:100]: can you actually incorporate this into the logger created in loggerP

File: packages/test/mocha-test-setup/src/mochaHooks.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ class TestLogger implements ITelemetryBufferedLogger {
         if (this.testName !== undefined) {
             event.testName = this.testName;
         }
+        event.hostName = "Mocha";
```
Comment (anthony-murphy) [inline packages/test/mocha-test-setup/src/mochaHooks.ts:16]: can you set this to the pkgName, it exists in packageVersion.ts. do this for both

File: packages/test/test-service-load/src/utils.ts (hunk 1 above)
Comment (chensixx) [reply packages/test/test-service-load/src/utils.ts:100]: Do you mean like adding an event property in FileLogger's send method?

Comment (anthony-murphy) [reply packages/test/test-service-load/src/utils.ts]: yes

File: packages/test/mocha-test-setup/src/mochaHooks.ts [hunk 3]
```diff
@@ -6,13 +6,15 @@
...
 import { Context } from "mocha";
+import { pkgName } from "./packageVersion";
 
 const _global: any = global;
 class TestLogger implements ITelemetryBufferedLogger {
     send(event: ITelemetryBaseEvent) {
         if (this.testName !== undefined) {
             event.testName = this.testName;
         }
+        event.hostName = `${pkgName}`;
```
Comment (anthony-murphy) [inline packages/test/mocha-test-setup/src/mochaHooks.ts:17]:
```suggestion
        event.hostName = pkgName;
```

File: packages/test/test-service-load/src/utils.ts [hunk 4]
```diff
@@ -58,6 +58,7 @@ class FileLogger extends TelemetryLogger implements ITelemetryBufferedLogger {
         this.baseLogger?.send(event);
 
         event.Event_Time = Date.now();
+        event.hostName = `${pkgName}`;
```
Comment (anthony-murphy) [inline packages/test/test-service-load/src/utils.ts:61]:
```suggestion
        event.hostName = pkgName;
```

**Review by anthony-murphy — APPROVED** [review]

File: packages/test/test-service-load/src/utils.ts [hunk 5]
```diff
@@ -58,6 +58,7 @@ class FileLogger extends TelemetryLogger implements ITelemetryBufferedLogger {
         this.baseLogger?.send(event);
 
         event.Event_Time = Date.now();
+        event.hostName = pkgName;
```
Comment (anthony-murphy) [inline packages/test/test-service-load/src/utils.ts:61]: actually, this should go above   this.baseLogger?.send(event);

Comment (anthony-murphy) [reply packages/test/test-service-load/src/utils.ts:61]: right now it won't go to telemetry

File: packages/test/test-service-load/src/utils.ts [hunk 6]
```diff
@@ -58,6 +58,7 @@ class FileLogger extends TelemetryLogger implements ITelemetryBufferedLogger {
         this.baseLogger?.send(event);
```
Comment (anthony-murphy) [inline packages/test/test-service-load/src/utils.ts:58]:
```suggestion
        this.baseLogger?.send({...event, hostName: pkgName});
```

File: packages/test/test-service-load/src/utils.ts (hunk 5 above)
Comment (anthony-murphy) [inline packages/test/test-service-load/src/utils.ts:61]:
```suggestion
```

File: packages/test/test-service-load/src/utils.ts (hunk 6 above)
Comment (anthony-murphy) [reply packages/test/test-service-load/src/utils.ts:58]: we don't need this to log in the file, as it's we know the host, as only stress logs to the file

**Pull request merged**